tests/cases/fourslash/autoCloseFragment.ts

```diff
@@ -50,7 +50,7 @@ verify.jsxClosingTag({
     1: undefined,
     2: undefined,
     3: undefined,
-    4: { newText: "</>" },
+    4: undefined,
```

Comment:
The case looks like:

```ts
// @Filename: /4.tsx
////const x = <div>
////    <>/*4*/
////    </div>
////</>;
```

So, we now see the fragment close below and don't offer up a close. Maybe reasonable, maybe not.